app/handlers/sessions_create.rb

```diff
@@ -225,11 +213,7 @@ def authentication_user
   end
 
   def signing_up?
-    pre_auth_state.present?
-  end
-
-  def pre_auth_state
-    options[:pre_auth_state]
+    nil
```

Comment:
hmm, instead of `nil`, maybe I should throw a Sentry message here so we can verify nobody is still getting through this path?

Reply:
sure

Reply:
done